contracts/UniswapV3Staker.sol

```diff
@@ -87,6 +87,8 @@ contract UniswapV3Staker is
             params.totalReward
         );
 
+        // TODO: use the named parameter constructor
```

Comment:
https://github.com/omarish/uniswap-v3-staker/issues/117
---^ check this issue title to make sure I understand correctly